### PR TITLE
DDF-6012 upgraded to Admin-UI 2.25.0

### PR DESCRIPTION
**Pull request by shaundmorris**
#### What does this PR do?
Upgrades to the released version of DDF Admin UI
This also will fix the release

#### Who is reviewing it? 
@blen-desta 

#### Select relevant component teams: 
@codice/build 
@codice/continuous-integration 
@codice/ui 

#### Ask 2 committers to review/merge the PR and tag them here.
@andrewkfiedler 

#### What are the relevant tickets?
Fixes: #6012 

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

---
This change is [Reviewable](https://reviewable.io/reviews/codice/ddf/6063)

### CODE REVIEW

**Comment by shaundmorris** [comment]
build now

File: pom.xml [hunk 1]
```diff
@@ -177,7 +177,7 @@
         <components-font-awesome.version>4.7.0</components-font-awesome.version>
         <countryconverter.version>0.1.8</countryconverter.version>
         <cxf.version>3.2.12</cxf.version>
-        <ddf-admin-ui.version>2.25.0-SNAPSHOT</ddf-admin-ui.version>
+        <ddf-admin-ui.version>2.25</ddf-admin-ui.version>
```
Comment (mojogitoverhere) [inline pom.xml:180]: Should this be 2.25.0?
```suggestion
        <ddf-admin-ui.version>2.25.0</ddf-admin-ui.version>
```

Comment (shaundmorris) [reply pom.xml]: yes, not sure how I managed that one. Thanks

**Comment by shaundmorris** [comment]
build now

**Comment by shaundmorris** [comment]
Merging to fix the release build.

**Pull request merged**